presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -181,6 +186,25 @@ public long getPlanNodeJoinProbeKeyCount()
         return planNodeJoinProbeKeyCount;
     }
 
+    public Optional<DynamicFilterStats> getDynamicFilterStats()
+    {
+        return dynamicFilterStats;
+    }
+
+    public static Optional<DynamicFilterStats> mergeDynamicFilterStats(Optional<DynamicFilterStats> stats1, Optional<DynamicFilterStats> stats2)
+    {
+        Optional<DynamicFilterStats> optionalDynamicFilterStats = Optional.empty();
+        if (stats1.isPresent()) {
+            DynamicFilterStats dynamicFilterStats = stats1.get();
+            stats2.ifPresent(dynamicFilterStats::mergeWith);
```

Comment:
What does stats2.ifPresent(dynamicFilterStats::mergeWith) do here? Thanks!

Reply:
It's to merge the stats of the same plan node reported by different tasks